tests/monitors/docker_container_stats/docker_container_stats_test.py

```diff
@@ -64,3 +72,20 @@ def test_docker_stops_watching_old_containers():
             time.sleep(5)
             backend.datapoints.clear()
             assert ensure_always(lambda: not has_datapoint_with_dim(backend, "container_id", nginx_container.id))
+
+
+@pytest.mark.k8s
+@pytest.mark.kubernetes
+def test_docker_container_stats_in_k8s(agent_image, minikube, k8s_test_timeout, k8s_namespace):
```

Comment:
I would just remove this test since docker isn't really a separate service running on k8s.